test/e2e/e2e-service-provider/src/main/java/org/apache/skywalking/e2e/lua/LuaController.java

```diff
@@ -36,8 +37,15 @@
     @PostMapping("/nginx/entry/info")
     private String nginxEntry(String backend) throws MalformedURLException, URISyntaxException {
         final URL url = new URL("http://nginx:8080/nginx/info");
+        TraceContext.putCorrelation("entry", "entry_value");
         final ResponseEntity<String> response = restTemplate.postForEntity(url.toURI(), null, String.class);
         return response.getBody();
     }
 
+    @PostMapping("/nginx/end/info")
+    private String nginxEnd() throws MalformedURLException, URISyntaxException {
+        return TraceContext.getCorrelation("entry").orElse("")
+            + "_" + TraceContext.getCorrelation("nginx").orElse("");
+    }
```

Comment:
> correlation can't be tested in the e2e from my perspective. 

@wu-sheng maybe here is how @mrproliu he made it possible to test correlation in the E2E, get the correlation context in a service and expose them in an endpoint that can be verified in the test codes?

Reply:
Here my changed, `java1` -> `Nginx` -> `java2`, `java1` and `Nginx` to put the correlation data, then I get the all correlation data in `java2` and return it. So, I make a JUnit to verify the `java2` result.

Reply:
`/nginx/end/info` is the validated endpoint name, but the response entity wouldn't be tests. Please correct me if I am wrong.